contrib/kube-prometheus/jsonnet/kube-prometheus/prometheus/prometheus.libsonnet

```diff
@@ -400,7 +398,7 @@ local k = import 'ksonnet/ksonnet.beta.3/k.libsonnet';
           },
           endpoints: [
             {
-              port: 'http-metrics',
+              port: 'metrics',
```

Comment:
we're selecting the `metrics` port, but the service doesn't specify this named port [0]. 

[0] https://github.com/coredns/deployment/blob/master/kubernetes/coredns.yaml.sed#L166-L172

Reply:
The pod does though: https://github.com/coredns/deployment/blob/bbf9284adc6bace3caf35a38a131baeda92dee18/kubernetes/coredns.yaml.sed#L122

Reply:
Nevertheless I think you're right it would make sense to add it to the service upstream.

Reply:
I've sent a PR to coredns/deployment to add the port to the service: https://github.com/coredns/deployment/pull/112